package-lock.json

```diff
@@ -23,6 +23,9 @@
         "@headlessui/react": "^1.7.14",
         "@mui/icons-material": "^5.11.16",
         "@mui/material": "^5.13.5",
+        "@ory/client": "^1.1.41",
```

Comment:
you probably have some command line that keep using `npm`? let's stick to yarn for now.

Reply:
These are older commits now we are using yarn.